scripts/qa_crowbarsetup.sh

```diff
@@ -2604,7 +2604,7 @@ function custom_configuration
             # don't deploy calamari by default on SOC7. calamari needs a postgres DB on localhost
             # and get's confused if it is deployed on the controller where a postgres DB is already running
             # see https://bugzilla.suse.com/show_bug.cgi?id=1008331
-            if iscloudver 7plus ; then
+            if iscloudver 6plus || iscloudver 7plus ; then
```

Comment:
6plus means SOC6 or later, so you do not need the 7plus then, as it is included in 6plus.